code/controllers/MailChimpController.php

```diff
@@ -148,10 +161,9 @@ public function McDoSubscribeForm($data, Form $form) {
 	 * @return Boolean
 	 */
 	public static function McSubscribe($email, $merge_vars = array()) {
-		
 		require_once MAILCHIMP . '/lib/MCAPI.class.php';
 		$api = new MCAPI(Config::inst()->get('MailChimpController', 'apikey'));
-		$retVal = $api->listSubscribe(Config::inst()->get('MailChimpController', 'listid'), $email, $merge_vars);
+		$retVal = $api->listSubscribe(Config::inst()->get('MailChimpController', 'listid'), $email, $merge_vars, $email_type='html', $double_optin=false, $update_existing=true, $replace_interests=true, $send_welcome=true);
```

Comment:
Sure, config option makes sense. Defaults are listed in MCAPI, line 1737. HTML is actualy on by default—however 'double opt-in', 'update existing' and 'send welcome' are all changed based on my needs.